2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -207,26 +214,30 @@ public void teleopInit()
       initialize();
     }
   }
-
   @Override
   public void teleopPeriodic()
   {
-    //driveTrain.DriveSystem(oi.pilot);
+    if(chassisEnable){    
+    driveTrain.DriveSystem(oi.pilot);
+    }
+    
     if(ImuEnable){
       imu.getvalues();
     }
-
-      //All spinner logic is in Spinner.java
-      spinner.spin(colorEnable);
-    
-
+    if(climberEnable){
+      climber.drive();
+    }
+    spinner.spin(colorEnable);
```

Comment:
It's a little surprising that this is not in an `if(colorEnable)` but since this isn't a spinner PR, let's leave it for now.